src/CollabVM.cpp

```diff
@@ -2271,6 +2285,20 @@ void CollabVMServer::OnListInstruction(const std::shared_ptr<CollabVMUser>& user
 		} else {
 			instr += "0.";
 		}
+	}else{ // this fucking sucks
```

Comment:
This area seems to imply you didn't run clang-format. Do that.

Another thing which I might mention is maybe having a separate BinaryResources header file which contains binary resources like this, so that it isn't embedded straight into code.